Task Liverpool/src/test/java/com/encora/automation/test/BaseTest.java

```diff
@@ -4,15 +4,20 @@
 import org.openqa.selenium.WebDriver;
 import org.testng.annotations.AfterSuite;
 import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeTest;
 
 public class BaseTest {
-
     protected WebDriver getDriver(){
         return DriverHandler.getWebDriver();
     }
 
     @BeforeTest
+    protected void openBrowserAndMaximize(){
```

Comment:
change this name to something more generic. If tomorrow you need to add more logic there it could be a problem